spec/paperclip/storage/s3_spec.rb

```diff
@@ -481,7 +481,7 @@ def original_filename
     before do
       rebuild_model (aws2_add_region).merge storage: :s3,
         s3_credentials: { bucket: "prod_bucket" },
-        s3_host_alias: Proc.new{|atch| "cdn#{atch.instance.counter % 4}.example.com"},
+        s3_host_alias: ->(atch) { "cdn#{atch.instance.counter % 4}.example.com" },
```

Comment:
Line is too long. [82/80]<br>Align the elements of a hash literal if they span more than one line.